packages/cli-helpers/package.json

```diff
@@ -25,7 +25,9 @@
     "@babel/runtime-corejs3": "7.20.6",
     "@redwoodjs/telemetry": "3.2.0",
     "core-js": "3.26.1",
-    "listr2": "5.0.6"
+    "listr2": "5.0.6",
+    "prompts": "2.4.2",
```

Comment:
I'll break this change (prompts) out of this PR and into https://github.com/redwoodjs/redwood/pull/7117 since it's just a bug fix.

Reply:
👍